helm/node-operator-chart/templates/rbac.yaml

```diff
@@ -43,45 +23,15 @@ rules:
       - serviceaccounts
     verbs:
       - get
-      - create
-      - delete
-      - list
-  - apiGroups:
-      - ""
-    resources:
-      - persistentvolumeclaims
-    verbs:
-      - get
-  - apiGroups:
-      - ""
-    resources:
-      - endpoints
-      - services
-    verbs:
-      - get 
       - list
       - watch
```

Comment:
I am not sure about these here. I would actually say the node-operator does not access clusterrolebindings and serviceaccounts directly. Is this necessary because of some inner workings? I now configured read access only, but would prefer to remove it completely if that makes sense.

Reply:
Yeah, it is good to start with calls you now the operator is doing, and debug logs about error in RBAC.
A useful way to make that is
`kubectl logs -l app=node-operator | grep serviceaccount`
And it gives errors like (e.g.)
`system:serviceaccount:giantswarm:node-operator cannot get configmaps in namespace ...`

Reply:
But what is the mechanism here? Do we have to define rules only for custom things our own code is doing or do we need to define rules for Kubernetes internals as well?

Reply:
You should do for your own code, but if you have a dependency (like I found storage in some operators) you have to allow requests from it. So what has worked for me is do for the operator itself and debug logs to find other request maybe are hidden in dependecies